docs/modeling/keys.rst

```diff
@@ -0,0 +1,67 @@
+.. include:: /rc1-notice.txt
+
+Keys (primary)
+==============
+
+A key serves as the primary unique identifier for each entity instance. When using a relational database this maps to the concept of a *primary key*.
```

Comment:
Could be useful to add a reference to the alternate keys page, something to the effect of:

    "You can also configure a unique identifier that is not the primary key. See `alternate keys`_"